docs/src/pages/overview/Overview.jsx

```diff
@@ -368,7 +368,7 @@ function Overview() {
             { text: "Facebook", href: "http://www.google.com" },
             { text: "Instagram", href: "http://www.google.com" },
           ]}
-          copyright="© DXC Technology 2020. All rights reserved."
+          copyright="© DXC Technology 2022. All rights reserved."
```

Comment:
Remove this line. This Footer (and every other one in our CDK must use the preset value).

Reply:
Done